src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPTO_ALGORITHM = "SHA-512";
+
```

Comment:
Util class should have a private constructor to prevent class instantiation.
Util class should be final.
Util class' methods should be static.